targetplatform/smarthome.target

```diff
@@ -152,6 +148,16 @@
 <unit id="californium-osgi" version="0.0.0"/>
 <unit id="org.eclipse.californium.element-connector" version="0.0.0"/>
 <unit id="org.eclipse.californium.scandium" version="0.0.0"/>
+<unit id="osgi.annotation" version="0.0.0"/>
```

Comment:
how is osgi.annotation related to mockito 2.0?

Reply:
Not at all - it was a follow-up on #4737 which was well overdue, which accidentally slipped into this branch here. Should have been done anyway, but you are right, it was intended for another PR.